ui/src/actions/actions-ui.js

```diff
@@ -52,6 +53,15 @@ export const toggleNavigationModal = (isOpen, targetLocation) => {
   };
 };
 
+export const toggleConfirmSaveAppModal = isConfirmSaveAppModalOpen => {
```

Comment:
@craicoverflow hadn't realised the renaming of `ConfirmSaveApp` to `SaveApp` touched so many places. I'd suggest keeping the naming consistent, and either removing or keeping `confirm` in the method and variable names associated with save-app-modal.